packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -48,40 +36,84 @@ async function main() {
   // In localhost network we allow to not set the emergency executor address and default to the first account.
   if (!process.env[EMERGENCY_EXECUTOR] && hre.network.name != "localhost") throw new Error("No emergency executor set");
 
-  if (!process.env[EMERGENCY_QUORUM]) throw new Error("No emergency quorum set");
-  const tenMillion = hre.ethers.utils.parseUnits("10000000", "ether");
-  const fiveMillion = hre.ethers.utils.parseUnits("5000000", "ether");
-  const emergencyQuorum = hre.ethers.utils.parseUnits(process.env[EMERGENCY_QUORUM], "ether");
-  if (emergencyQuorum.gt(tenMillion) || emergencyQuorum.lt(fiveMillion)) throw new Error("Invalid emergency quorum");
+  // Start DVM2.0 parameters
+  const emergencyQuorum = hre.ethers.utils.parseUnits("5000000", "ether");
+  const emergencyExecutor = process.env[EMERGENCY_EXECUTOR] || (await hre.ethers.getSigners())[0].address;
```

Comment:
What address will this be set to? We don't intend to set it to the deployer, right?

I might suggest removing the fallback unless we really need it to avoid unintentional misconfiguration.

Reply:
after looking at this this does not work due to the transfer call within the constructor of the EmergencyProposer [here](https://github.com/UMAprotocol/protocol/blob/md0x/votingv2-upgrade-updates/packages/core/contracts/data-verification-mechanism/implementation/EmergencyProposer.sol#L117)